source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -0,0 +1,2163 @@
+{
```

Comment:
<blockquote>The following example need to be run on a machine with at least one NVIDIA GPU.</blockquote><p>Instead of saying we need a GPU we can say something like "Wiith GPUs we can do the following workflow, and we don't even need that GPU locally because Databricks can provide that for us.".</p><p> </p><p>But generally I think here we need to go for a bit more of a cook book style and break things down into a couple of steps:</p><ul><li>dask-databricks can help us launch a Dask cluster alongside Spark on databricks</li><li>Create a new init script that installs RAPIDS and runs dask-databricks</li><li>Create a new multi-node cluster that uses the script</li><li>Once the cluster is running upload this notebook to Databricks and continue running these cells on there</li></ul><p> </p><p>We should also use the [<code>docref</code> admonition](<a href="https://github.com/rapidsai/deployment#docref" target="_blank">https://github.com/rapidsai/deployment#docref</a>) here to better cross-link to the Databricks page. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>